src/implementations/vanilla/utilities/typography/typography.scss

```diff
@@ -0,0 +1,173 @@
+@use "sass:map";
+@use "@ecl/theme-dev/maps/typography";
+@use "@ecl/theme-dev/maps/color";
+@use "@ecl/theme-dev/maps/layout";
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
+/*
+   * Headings
+   */
+.ecl-u-type-heading-1 {
```

Comment:
I think that the font values for h1 to h3 are wrong here.
Could you double check?

Reply:
Thanks for noticing, there were indeed a few differences